\documentclass{bioinfo}
\copyrightyear{2014}
\pubyear{2014}




%%% Additional Macro and command.
\usepackage{url}
\newcommand{\pkg}[1]{{\fontseries{b}\selectfont #1}}
\makeatletter
\newcommand\code{\bgroup\@makeother\_\@makeother\~\@makeother\$\@codex}
\def\@codex#1{{\normalfont\ttfamily\hyphenchar\font=-1 #1}\egroup}
\makeatother
\let\proglang=\textsf
%%% End of additional Macro and command.


\newcommand{\package}{ribModel } % need that whitespace there for text flow


%%% Start here.
\begin{document}
\firstpage{1}

\title[ribModel]{ribModel: an \proglang{R} package for codon usage bias fits}
\author[
Landerer \textit{et~al}]{Cedric Landerer\,$^{1,3}$\footnote{
to whom correspondence should be addressed
},
Alex Cope\,$^{1,4}$
Russell Zaretzki\,$^{2,3}$, and
Michael Gilchrist\,$^{1,3}$
}
\address{$^{1}$
Department of Ecology and Evolutionary Biology,
$^{2}$Department of Statistics, Operations, and Management Science, and
$^{3}$National Institute for Mathematical and Biological Synthesis,
University of Tennessee, Knoxville, TN, USA,
$^{4}$Oak Ridge National Labratory, Oak Ridge, TN, USA} 
\history{Received on XXXXX; revised on XXXXX; accepted on XXXXX}

\editor{Associate Editor: XXXXXXX}

\maketitle

\begin{abstract}

\section{Summary:}
\pkg{\package} is a collection of codon models, estimating terms related to mutation and selection coefficients of synonymous codon usage bias (CUB) and population genetics parameter of interest. 
The implemented models allow to analyze sequence data as well as ribosome foot printing data to estimate selection on ribosome overhead cost, nonsense error rate and ribosome pausing times. 
\package also allows for the estimation of the evolutionary average protein synthesis rate, reflecting the environmental conditions an organisms evolves under. 
\package implements a Metropolis-Hastings withing Gibbs sampling to fit the codon models.
The framework provides an R interface for ease of use and is implemented with a generic design to allow for future additions of additional codon models.

\section{Availability:}
\pkg{\package} and documents are freely available under the Mozilla Public License 2.0
on CRAN (\url{http://cran.r-project.org/package=cubfits}).

\section{Contact:} \href{cedric.landerer@gmail.com}{cedric.landerer@gmail.com}
\end{abstract}


%\section*{Introduction}

%\begin{itemize}
%\item Huge influx of genome scale data sets across the tree of life
%\item Why study CUB?
%	\begin{itemize}
%	\item CUB is shaped by mutation and selection
%	\item informs about selection on translation efficiency.
%	\item translation efficiency can mean ribosome pausing, error free translation (nonsense or missense errors).
%	\item CUB potentially relates to co-translational folding. Tools allows to get at questions
%	\end{itemize}
%\item compare to CAI and tAI
%	\begin{itemize}
%	\item CAI and such require reference set of house-keeping gens for comparison of adaptation
%	\item tAI uses tRNA copy number implying no other factors are involved
%	\item We have population genetics component getting at $N_e * s$ with our model
%	\end{itemize}
%\item inference of expression as evolutionary mean unaffected by growth conditions and experimental noise
%\item Bayesian MCMC method allows for priors and multilevel and hierarchy 
%\end{itemize}



%\section*{Results}
%\begin{itemize}
%\item Performance (simulated data)
%	\begin{itemize}
%	\item runtime by number of genes and number of mixtures
%	\item confidence in parameters by number of genes
%	\item estimation of phi values (gene ordering)
%	\end{itemize}
%\item $s_{epsilon}$ as a measure of noise due to the lab conditions by eliminating technical 			noise. Use replicates to estimates technical noise
%\end{itemize}


\section*{Introduction}
Driven by improvements in DNA sequencing technology and methodology, the past 15 years has seen an exponential growth in the number of publicly available genomes. 
This influx of genomic data necessitated the development of computational tools which allow researchers to extract information.

Information about selection on the translation process, such as factors shaping translational efficiency and co-translational folding, can be extracted using codon usage bias.
CUB refers to non-uniform usage of synonymous codons shaped by bias in mutation and selection.
Although CUB was first studied almost 40 years ago, many questions related to the factors shaping CUB remain open.
As a result, many metrics were developed in an attempt to quantify CUB, such as Codon Adaptation Index (CAI).
There are many issues with these metrics.
For example, CAI relies on a reference set of genes assumed to be highly expressed, such as genes coding for ribosomal proteins.
Most of these metrics only account for selection in shaping CUB and is not rooted in population genetics. 
Recent work demonstrated that to accurately quantify CUB, mutation bias and genetic drift are also needed.
In [Gilchrist et al 2015], a mechanistic model rooted in population genetics that accounted for selection, mutation bias, and genetic drift was presented. 
Here, we describe an open-source computational tool that allows researchers to analyze CUB data using the model presented in [Gilchrist et al 2015].
In addition this work expands Gilchrist et al 2015 by allowing for mixture distributions for mutation and selection within a dataset.

Information on mutational and selection processes can be obtained from codon usage bias.
 
The influence of mutation and selection is related to the evolutionary averge protein production rate $\phi$ of a gene.
Genes with increased protein synthesis rate are belived to be under higher selection for translation efficiency and are therefore composed of a higher percentage of optimal codon.
Gene regions that deviate from that pattern can indicate to explore differential selection on codon usage for the purpose of co-translational folding or signal peptides


\begin{figure*}[!tpb]
\centering
 \includegraphics[width=3.5in]{expl_model.png}
\vspace{-0.2cm}
\caption{Binning for amino acid Isoleucin by expression levels (y-axis) where dots are mean proportions of synonymous codons (x-axis) of 100 simulated sequences in every 5\% expression windows, and vertical lines are 90\% empirical intervals. The curves are theoretical prediction of codon usages.
}
\label{fig:plotbin}
\end{figure*}

\section*{Features}
\package provides an easy to use collection of codon models to analyze genome scale data such as gene sequences and ribosome foot printing data and estimate relevant population genetic terms. 
The provided models analyze sequence data based on their codon usage bias and allow for the estimation of mutation bias and selection for translation efficiency or selection against nonsense errors. 
Ribosome foot printing data can be analyzed inferring ribosome pausing times.  
The provided framework was designed to allow researchers to analyze their data swiftly without the need for long pre-processing of data sets. 

\subsubsection*{Mixture distributions}
\package also extends the implemented codon models by allowing for a mixture distribution in mutation and selection terms.
This feature makes \package ideal to answer questions about intra-genomic heterogeneity in mutation and selection like they might occur during hybridization events or within gene heterogeneity due to co-translational folding or signal peptides.

\subsubsection*{Writing extensions}
\package provides a general framework, designed to allow for simple extensions, providing new models to analyze genome scale data. 
The framework follows the classical design principles of object orientation and inheritance (Booch 1993). 
An object oriented design was chosen to encapsulate models and allow them to share commonalities to allow the fast implementation of new codon models. 
Researchers developing new codon models are invited and welcome to incorporate them into the provided framework.

\subsubsection*{C++ to aid with the heavy lifting}
\package is implemented completely in C++ to improve performance and allow for a generic design.
However, R does not provide a native C++ support, therfore we utilized the R package Rcpp (Eddelbuettel \& Francois 2011). 
Rcpp provides a module structure allowing the exposure of whole C++ classes to R thus minimizing data transfer between the R environment and the C++ core and improving performance. 
The runtime of \package scales like expected linearly with genome size and iterations and XX with the increase of mixture distributions in the data set.

\section*{Discussion}
\begin{itemize}
\item Runtime, scalability
\item Extendability, FONSE, PANSE, RFP
\item applications, hybridization, phylogenetics, variation in mutation and selection
\end{itemize}



%\bibliographystyle{natbib}
%\bibliography{bioinfo}
\end{document}
