\chapter{Is thermosensing property of RNA thermometers unique?} \label{ch:rna.thermo}

This chapter is a lightly revised version of a paper by the same name published in PLoS ONE and co-authored with Michael A. Gilchrist.\\
\newline
\newline
Shah and Gilchrist. Is thermosensing property of RNA thermometers unique?. PLoS ONE (2010) vol. 5 (7) pp. e11308

\clearpage
\pagebreak

\section*{\centering Abstract}
A large number of studies have been dedicated to identify the structural and sequence based features of RNA thermometers, mRNAs that regulate their translation initiation rate with temperature. 
It has been shown that the melting of the ribosome-binding site (RBS) plays a prominent role in this thermosensing process. 
However, little is known as to how widespread this melting phenomenon is as earlier studies on the subject have worked with a small sample of known RNA thermometers. 
We have developed a novel method of studying the melting of RNAs with temperature by computationally sampling the distribution of the RNA structures at various temperatures using the RNA folding software Vienna. 
In this study, we compared the thermosensing property of 100 randomly selected mRNAs and three well known thermometers - \emph{rpoH}, \emph{ibpA} and \emph{agsA} sequences from \emph{E.~coli}.
We also compared the \emph{rpoH} sequences from 81 mesophilic proteobacteria.
Although, both \emph{rpoH} and \emph{ibpA}, show a higher rate of melting at their RBS compared with the mean of non-thermometers, contrary to our expectations these higher rates are not significant. 
Surprisingly, we also do not find any significant differences between \emph{rpoH} thermometers from other $\gamma$-proteobacteria and \emph{E.~coli} non-thermometers.

\clearpage
\pagebreak

\section{Introduction}
Many microorganisms live in a variable environment. 
They have evolved a variety of mechanisms to sense changes in their environment and alter their gene expression in response to these changes. 
Regulatory proteins often play a role in controlling the level of transcription and translation of other genes. 
However, in certain cases post-transcriptional mechanisms, such as changes in mRNA conformation, are known to influence gene expression. 
In some prokaryotes, reaction to changes in the temperature is thought to be mediated by one such class of mRNAs called RNA thermometers \citep{YuzawaEtAl93, NakahigashiEtAl95, MoritaEtAl99, ChowdhuryEtAl03, NarberhausEtAl06}.
At lower temperatures, the thermosensing region in these sequences adopts a secondary structure that sequesters the ribosome binding site (RBS) of a gene, hence interfering with translation initiation by the ribosome. 
At higher temperatures, this thermosensing region upstream of the coding sequence ‘melts’, increasing the accessibility of the RBS leading to an increase in the initiation of translation and, in turn, its protein production rate \citep{DeSmitAndVanDuin90, YuzawaEtAl93, ChowdhuryEtAl03, NarberhausEtAl06}.

Previous work on RNA thermometers has focused primarily on understanding and identifying their sequence based features and residues important for thermosensing \citep{DeSmitAndVanDuin90, YuzawaEtAl93, NakahigashiEtAl95, MoritaEtAl99}. 
Time elapsed spectral studies \citep{ChowdhuryEtAl06} and mutational analyses \citep{YuzawaEtAl93, NakahigashiEtAl95, MoritaEtAl99} of the thermometer genes have been used to identify regions, which play a crucial part in the thermosensing property. 
For instance, in one of the most studied RNA thermometer called the ROSE (Repression Of heat-Shock gene Expression) element, a guanine residue at position 83, paired opposite the Shine-Dalgarno (SD) sequence in a hairpin structure is known to play a prominent role in the ability of the mRNA to change its expression with temperature \citep{ChowdhuryEtAl03}.

Although these studies provide insights into the mechanisms by which specific thermometers function, little is known as to how widespread these mechanisms are. 
The fraction of genes in a genome that possess an ability to regulate their translation by thermosensing or a similar mechanism is unknown. 
More importantly, because the above studies do not include non-thermometers as controls, it is difficult to ascertain if RNA thermometers are a special class of molecules different from other RNAs. 
Since it is not feasible to perform mutational or spectral studies on every gene to identify whether it behaves as an RNA thermometer, computational tools need to be developed to provide these insights. 
We here propose a computational approach to characterize RNA thermometers and ask how they differ from non-thermometers in their ability to melt with an increase in temperature. 
Understanding the melting potential of non-thermometers should aid in understanding the adaptive features of RNA thermometer sequences. 
We focus specifically on the ability of genes to change their expression by modifying the accessibility of RBS, or in other words, `RBS exposure'. 

Earlier attempts to identify potential RNA thermometers have focused on search patterns based on similarities in the secondary structure of the mRNAs \citep{WaldminghausEtAl05, WaldminghausEtAl07}. 
However, the use of a fixed length sequence for secondary structure limits the utility of this approach.  
For instance, sequences that differ by only a single nucleotide in their lengths can have drastic differences in their predicted secondary structures \citep{HughesAndMcElwaine06}.
Secondly, most studies when looking at secondary structures of RNAs use mainly the least free energy (LFE) structures. 
Although, this approach of using the most stable structures has proved useful, there are certain shortcomings when used for characterizing RNA thermometers.
It has been shown that as temperature increases, the overall probability and uniqueness of finding a structure in its LFE state decreases \citep{HuynenEtAl97, VossEtAl04}.
Thus, such an approach could lead to spurious results as the energy landscape of the molecule evolves with temperature (Fig. \ref{fig:landscape}). 
In addition, looking at LFE structures at a single temperature alone provides no means of quantifying the effect of temperature on the structure. 
Finally, any pattern-based approach to finding thermometers is restrictive, as it does not take into account novel structures that might be thermosensing. 

\begin{figure}[H]
\begin{center}
\includegraphics[width=6.5in]{../Figure/c3Fig_1.pdf}
\end{center}
\caption{
{ Effect of temperature on the energy landscape.}}
As temperature increases, the probability of finding an mRNA in its most stable state decreases. 
This is because at higher temperatures, molecules have more energy enabling them to spend more time in higher energy states. 
Also, at higher temperatures, as the energy landscape becomes flatter, uniqueness of the stable state may also be lost \citep{HuynenEtAl97}.

\label{fig:landscape}
\end{figure}


Here we propose a novel method of quantitatively studying secondary structures of RNAs that addresses all of the above shortcomings. 
This method explores the ability of mRNAs to change their rate of translation initiation with temperature. 
We see this approach as complementary to experimental studies in the field of RNA structures. 

\section{Methods}
We used the RNAsubopt package from RNA folding software Vienna \citep{HofackerEtAl94} to predict secondary structures of the RNAs. 
This package was used to sample 1000 secondary structures at each temperature for every gene from the entire distribution of structures at that temperature. 
The sampling of sub-optimal structures is important because RNA secondary structures with very similar free energies can have drastic differences in their secondary structures \citep{VossEtAl04}, which might not be captured when looking at the structure with least energy in isolation. 
The program RNAsubopt generates structures with \emph{probabilities equal to their Boltzmann weights} via stochastic backtracking in the partition function \citep{WuchtyEtAl99}.
Since these structures are drawn based on their Boltzmann weights, the entire ensemble of 1000 structures can be viewed as a time ensemble, i.e., the probability of finding a particular structure in our ensemble is proportional to the amount of time the RNA is found to be in that structure. 
Thus, stable structures would have higher Boltzmann weights and the RNA would spend a greater amount of time in that structure.

In order to understand the effect of temperature on gene expression as measured by RBS exposure, we randomly selected 100 non-thermometer mRNAs from the \emph{E.~ coli} genome \citep{ShahAndGilchrist10a} as well as \emph{rpoH} mRNA sequence, a known thermometer, from 81 mesophilic $\gamma$-proteobacteria for this study \citep{ShahAndGilchrist10a}. 
Transcript start and end positions for \emph{E.~coli} genes were obtained from the RegulonDB database \citep{SalgadoEtAl06}. 
Information regarding the position of RBS on the transcript was obtained from the \emph{flexrbs} dataset \citep{ShultzabergerEtAl01,ShahAndGilchrist10a}.
We used the entire length of the mRNA ($5'$ UTR + ORF + $3’$ UTR) to generate the sub-optimal structures.
This was done for the following reasons. 
The secondary structure of mRNA is highly dependent on the length of the sequence used for simulation \citep{HughesAndMcElwaine06}. 
Using a shorter length may prevent detection of any long-range interactions that might be crucial for the stability, and function of the RNA molecule. Moreover, although translation is coupled with transcription in prokaryotes, the half-life of an mRNA is considerably longer than the time required for translation \citep{BernsteinEtAl02, HambraeusEtAl03, SelingerEtAl03} and hence the mRNA transcript would spend most of its time as a full-length sequence.
Thus, we argue that the secondary structure of the mRNA is better simulated by using the entire mRNA length for our purposes.
We also check whether our results are robust to using an mRNA sequence of length 150 nucleotides centered around the RBS.
Of the 100 genes from \emph{E.~coli}, 56 genes were part of operons. 
In the case of operons, we simulated the entire mRNA sequence but categorized multiple RBSs within an operon individually. 

We simulated 1000 secondary structures of each mRNA at 7 different temperatures ranging from 25 $^\circ$C to 50 $^\circ$C. 
All other parameters in RNAsubopt were used at default values. 
In order to quantify the openness of RNA, we used a sliding window length of 7 bases to estimate the fraction of simulated structures in which none of the bases in that window were involved in base pairing. 
A window length of 7 was chosen because the Shine-Dalgarno sequence/RBS in \emph{E.~coli} varies from 4-7 bases \citep{ShultzabergerEtAl01, Kozak05}. 
Changing the window length from 5 bases to 10 bases still resulted in the same qualitative behavior. 
However, as one would expect, because of the categorical nature of the data (open or close), the fraction of open or ‘melted’ windows in the structure decreased with window length.

An alternative to sampling structures based on Boltzmann's distribution is to estimate the least free energy (LFE) structures by constraining the RBS in the open conformation \citep{MathewsEtAl04}.
The LFE of the constrained and the unconstrained structures can then be used to estimate probability of openness of the RBS.
However, as mentioned earlier, with and increase in temperature, the overall probability and uniqueness of finding a structure in its LFE state decreases \citep{HuynenEtAl97, VossEtAl04}.
Thus, such a method severely limits the ability to compare the probability of openness across temperatures.

In order to compare the probability of openness across temperatures, we fitted a logistic model to the fraction of open windows as a function of temperature. 
\begin{equation}
\label{eq:logistic}
p_i(T)=\frac{e^{a_i+b_iT}}{1+e^{a_i+b_iT}}
\end{equation}
where $p_i(T)$ is the probability of finding the window at position $i$ in a gene, open at temperature $T$ ($^\circ$C), $a_i$ and $b_i$ are the intercept and slope parameters of how the log-odds of finding an open window at position $i$, $log(\frac{p_i(T)}{1-p_i(T)})$ , changes with temperature. 
The ratio $-a_i/b_i$ indicates the temperature at which the probability of openness of a window is 0.5. 
Although the probability of openness of RBS is positively correlated with protein expression, the exact relationship between the two is unknown. 

We find that the logistic model serves as a reasonable descriptor of RNA melting (Fig. \ref{fig:logistic}).

\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_2.pdf}
\end{center}
\caption{
{Fitting logistic regression.}}
The solid circles indicate the probability of openness, $p_i$ at the RBS of \emph{rpoH} gene. 
The open circles and squares represent two randomly chosen windows within \emph{rpoH}. 
The best fit lines of the logistic regression are given by the solid line for RBS and dashed and dotted line for the randomly chosen windows.

\label{fig:logistic}
\end{figure}


At very low temperatures, we expect most of the bases in the RNA to be paired with other bases. 
Hence, the probability of openness of a window would approach 0. At very high temperatures, the free energy of base-pairing decreases and most bases would be unpaired causing the probability of openness to approach 1. 
Thus in a specific range of temperatures, determined by the parameters $a$ and $b$, we can potentially see a transition between the two states.
However, we restrict our simulations to the biological relevant temperature range for mesophiles (25 $^\circ$C - 55 $^\circ$C).
In this study, we are primarily interested in the parameter $b$, which describes the rate of change of openness with temperature. 
For each window within each gene, the Maximum-Likelihood Estimates (MLE) of $a$ and $b$ were calculated using R \citep{Rcran08}. 

\section{Results}
\subsection{Capturing the behavior of RNA thermometers}
To show that our method is capable of capturing the increase in openness of the RBS of an RNA thermometer, we used the \emph{rpoH} gene sequence of \emph{E.~coli}. 
The \emph{rpoH} gene is a $\sigma$-factor involved in the up-regulation of the heat-shock proteins during higher temperatures. 
It is one of the most studied RNA thermometers \citep{YuzawaEtAl93, NakahigashiEtAl95, MoritaEtAl99}. 
Fig. \ref{fig:rbs.openness} illustrates how as temperature increases, the RBS of \emph{rpoH} shows a much higher fold-change in openness as compared to the regions flanking it. 
The openness of the RBS at 50 $^\circ$C was 25 folds higher than at 25 $^\circ$C. 
These results are consistent with the idea that the RBS of a gene might be under stronger selection to increase its openness with temperature.

\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_3.pdf}
\end{center}
\caption{
{ Fold-change in the openness of the RBS and regions 5 bases upstream and downstream of it with temperature.}}
The fold change is with respect to the openness at 25 $^\circ$C. The RBS of \emph{rpoH} gene has a much higher increase in openness with temperature than the regions around it.

\label{fig:rbs.openness}
\end{figure}

%Because the quantitative relationship between openness of the RBS and the corresponding protein expression is unclear, we use openness of RBS as a proxy when making comparisons between genes. 
%In order to show that our method was capable of capturing the observed behavior of 
We were also able to replicate the experimental results of \citet{WaldminghausEtAl05} where they showed that the deletion of guanine at position 71 (G71) of the gene \emph{ibpA} in \emph{E.~coli}, resulted in a loss of thermosensing activity. 
Fig. \ref{fig:ecoli.rbs.b} shows that both the RNA thermometers \emph{rpoH} and \emph{ibpA} have a higher rate of increase in their RBS exposure compared to the mean of the randomly selected 100 \emph{E.~coli} genes.
However, the MLE of $b$ drops to 0 when G71 is removed from the \emph{ibpA} gene sequence, as we did not observe a single open window in 1000 runs at all temperatures between 25 $^\circ$C and 50 $^\circ$C at that position.

\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_4.pdf}
\end{center}
\caption{
{The distribution of MLE estimates of $b$ of the 76 genes that differed significantly from zero in \emph{E.~coli}.}}
\emph{rpoH}, \emph{ibpA} and \emph{agsA} genes show an increase in openness with temperature with $b$ values 0.213, 0.295 and 0.042, respectively. 
However, none of these values are significantly higher than the mean of the distribution (\emph{Wilcox test}, $p\text{-value} = 0.156,\ 0.066\text{ and }0.945$, respectively). 
In addition, when the base G71 is removed from \emph{ibpA} sequence, the MLE estimate of $b$ reduces to 0.

\label{fig:ecoli.rbs.b}
\end{figure}


\subsection{Comparing thermometers and non-thermometers}
When the rate of openness of RBS, $b$, was compared across the 100 genes, we found that $b$ values were not significantly greater than zero for 24 genes at $p~value = 0.05$. 
This implies that a small fraction of genes did not show a significant change in openness of its RBS with temperature over the range of temperatures considered. 
This is surprising because if RNA thermometers were a rare class of mRNAs, then this number would have been far higher.
The distribution of the $b$ values for the remaining 76 genes is shown in Fig. \ref{fig:ecoli.rbs.b}.
Since the distribution of $b$ values is not a Gaussian distribution (\emph{Shapiro-Wilk test}, $p\text{-value}<10^{-5}$), non-parametric tests were employed for further statistical analyses.
Although the two of the three RNA thermometers, \emph{rpoH} and \emph{ibpA} had a higher $b$ value than the mean of the entire distribution ($\bar{b}=0.157$), these higher rates of openness were not significant (\emph{Wilcox test}, $p\text{-value} =0.156$ and $p\text{-value}=0.066$, respectively).
Interestingly, we find that RNA thermometer \emph{agsA} had a $b=0.042$, which, although positive, is lower than the mean of the distribution of $b$ values of non-thermometers.
We also show that there is no qualitative difference in our results when considering only 150 nucleotides of the mRNA centered around the RBS (see Figure \ref{sfig:b.150.dist}).


\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_S1.pdf}
\end{center}
\caption{
{The distribution of MLE estimates of $b$ of the 75 genes that differed significantly from zero in \emph{E.~coli}.}}
\emph{rpoH}, \emph{ibpA} and \emph{agsA} genes show an increase in openness with temperature with $b$ values 0.109, 0.137 and 0.0, respectively.
However, none of these values are significantly higher than the mean ($\bar{b}=0.158$) of the distribution (\emph{Wilcox test}, $p\text{-value} = 0.781,\ 0.500\text{ and }0.958$, respectively). 
In addition, when the base G71 is removed from \emph{ibpA} sequence, the MLE estimate of $b$ reduces to 0.

\label{sfig:b.150.dist}
\end{figure}

This result did not change even after including non-significant values of $b$ in the above test.
This indicates that RNA thermometers do not differ significantly from non-thermometers in increasing the openness of RBS with temperature. 
It argues that every RNA molecule has an inherent tendency to melt with temperature, albeit to varying degree. 
These results are also consistent when considering the window spanning the start codon (ATG) (see Fig. \ref{fig:ecoli.atg.b}), stability of which has been shown recently to be correlated with gene expression \citep{KudlaEtAl09}.

\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_5.pdf}
\end{center}
\caption{
{The distribution of MLE estimates of $b$ at the start codon (ATG) of the 85 genes that significantly differ from zero in \emph{E.~coli}.}}
\emph{rpoH}, \emph{ibpA} and \emph{agsA} genes show an increase in openness with temperature with $b$ values 0.095, 0.084 and 0.076, respectively. 
However, all the values are less than mean of the distribution. 
Also, when the base G71 is removed from the \emph{ibpA} sequence, the MLE estimate of $b$ reduces to 0.054. 
The results are consistent with what is observed at the RBS window.

\label{fig:ecoli.atg.b}
\end{figure}

Interestingly, the median transition temperature, given by $-a/b$, was $\sim68~^\circ$C.
Although the majority of the transition temperatures lie outside the temperature range experienced by mesophiles, it is important to note that this temperature indicates when the probability of openness is 0.5.
Although, the relationship between degree of openness and translation initiation is positively correlated, there exists no quantitative estimate of this relationship.
The above values indicate that the RBS needs to be open only a small fraction of time for translation initiation of most genes to meet their target protein production rates.

In order to show the generality of the above results, we compared the distribution of $b$ of \emph{rpoH} of 81 mesophilic $\gamma$-proteobacteria to that of the 100 randomly selected genes. 
Surprisingly, of the 81 \emph{rpoH} sequences, 17 (~21\%) showed no significant change in their $b$. 
We also found that the mean of the two distributions are not significantly different from each other (\emph{Kolmogorov-Smirnov test} $p\text{-value} =0.794$), further supporting our conclusions. 
Fig. \ref{fig:proteo.rbs.b} shows the distribution of 76 \emph{E.~coli} genes with significant $b$ values alongside the significant $b$ values of \emph{rpoH} genes of 64 mesophilic $\gamma$-proteobacteria. 

\begin{figure}[H]
\begin{center}
\includegraphics[width=4in]{../Figure/c3Fig_6.pdf}
\end{center}
\caption{
{ Distribution of significant $b$ values of 76 \emph{E.~coli} genes and 64 \emph{rpoH} genes of mesophilic $\gamma$-proteobacteria.}}
The two distributions are not significantly different from each other (\emph{Kolmogorov-Smirnov test} $p\text{-value} =0.794$).

\label{fig:proteo.rbs.b}
\end{figure}


\section{Discussion}
We present here a novel method of studying the melting of RNAs with temperature by incorporating the entire distribution of the RNA structures at a given temperature. 
This approach is more holistic as it takes into account the probability of finding the RNA in a sub-optimal structure based on its free energy as opposed to previous studies which have looked at structures with the least free energies only \citep{AvihooAndBarash05, MoritaEtAl99, NakahigashiEtAl95, WaldminghausEtAl05, WaldminghausEtAl07}. 
Although using the minimum free energy structure makes the analyses of structural features easier, it ignores the sub-optimal, yet highly likely structures that the RNA molecule can also adopt. 
Using the minimum free energy structure also becomes progressively problematic with and increase in temperature. 
It has been shown that as temperature increases, probability of finding the RNA in the minimum free energy structure becomes smaller \citep{HuynenEtAl97} as at higher temperatures, various secondary structures become equally probable as the energy landscape becomes shallower and flatter. 
Thus, for RNAs whose structure changes with temperature, it becomes important to sample from the entire distribution of structures. 
In addition, since our approach is not biased towards any particular structural feature, it can be used to identify novel thermosensitive structures. 

As one would expect, we find that mRNAs have an inherent tendency to ‘melt’ with an increase in temperature. 
This tendency varies with the sequence and the difference in temperatures. 
Contrary to our expectations, we find that RNA thermometers are not unique with respect to their ability to increase their RBS exposure with temperature. 
Since it is difficult and expensive to demonstrate the effect of temperature on the RNA secondary structure in the laboratory, researchers have focused primarily on known RNA thermometers. 
However, due to a lack of such studies on non-thermometers, it has been hard to ascertain whether thermosensing properties are unique to a ‘special’ class of RNAs.
Our results call for further experimental exploration of `non-thermometers' with changes in temperatures, before firm conclusions can be drawn regarding the uniqueness of RNA thermometers. 

Physiological similarities between RNA thermometers and non-thermometers with respect to their melting with temperature, raise an important question that if a large number of mRNAs show an extensive increase in RBS exposure with temperature, why don't we see corresponding changes in their protein expressions.
In other words, why do physiological similarities not lead to functional similarities?
This discrepancy could be explained, in part, by the fact that the amount of protein expression depends on a variety of factors such as mRNA abundance and stability, amount of regulatory proteins, the stability of the protein itself, and factors apart from the accessibility of the RBS of the mRNA to the ribosome. Hence, although temperature may not result in significant phenotypic effects of certain genes in terms of protein expression, it does not preclude the possibility of changes in its RBS exposure. 
Thus, the above results indicate that increased RBS exposure does not solely define as to what constitutes an RNA thermometer.

One of the key challenges in such studies is to devise appropriate measures that quantify the structural features in analyzing the distribution of secondary structures. 
Here, we use a simple measure of openness to quantify the changes in the structure with temperature. 
In order to quantify complex structural features like stems and loops in a distribution of RNA structures, more sophisticated measures could be developed. 
Our analysis based on the current state of RNA folding algorithms is also limited by the simple energy model as well as parameter estimates used in most algorithms. 

Another key limitation of this study is the fact that current RNA folding algorithms do not take into account the effect of presence of ribosome on the mRNA’s secondary structure. 
The secondary structure of an mRNA becomes a constantly changing environment due to the presence and movement of ribosomes along the mRNA affecting the openness of a window both upstream and downstream of its current position. 
Hence, including the effect of ribosomes on the mRNA on translation initiation in the folding algorithm may be important in identifying RNA thermometers computationally. 
This is likely to be a non-trivial task both mathematically and computationally.
However, we believe that incorporating the movement of ribosomes in RNA folding routine would open new avenues of research in investigating and understanding not only the effect of ribosome on the RNA structure and in translation initiation but also on the effect of any RNA-protein interactions on the secondary structure of the RNA. 
% You may title this section "Methods" or "Models". 
% "Models" is not a valid title for PLoS ONE authors. However, PLoS ONE
% authors may use "Analysis" 
%\section*{Materials and Methods}
%\pagebreak
% Do NOT remove this, even if you are not including acknowledgments
\section{Acknowledgments}
Funding for this project was provided by the Department of Ecology and Evolutionary Biology at the University of Tennessee, Knoxville.
P. S. was additionally supported by National Institute for Mathematical and Biological Synthesis (NIMBioS) Research Assistantship.

\clearpage
\pagebreak