\chapter{Conclusion} \label{ch:conclusion}

\section{Synthesis}
Over the years, a number of factors have been proposed to explain specific patterns of codon usage bias (CUB).
Thus, the problem of understanding these patterns is not of identifying potential evolutionary forces but that of estimating their relative importance \citep{ShahAndGilchrist10b,PlotkinAndKudla11}.
One of the main challenges in understanding the role played by various selective forces in shaping CUB lies in the fact that there exists no coherent framework to test these hypotheses.
The majority of the work on CUB has been correlative and focussed on using heuristic indices to quantify the bias \citep{BennetzenAndHall82,SharpAndLi87,Wright90}.
While heuristic approaches play an important role in exploring datasets, especially in the initial stages of analysis, the lack of mechanistic principles sheds little light on cause and effect
Moreover, since heuristic indices are based on individual researchers' intuition, they can lead to contradictory results depending on the index used \citep{StoletzkiAndEyreWalker07,GilchristEtAl09}.

In contrast, building upon the insights developed in \citep{GilchristAndWagner06,Gilchrist07}, we have developed a robust framework of incorporating mechanistic models of protein translation into classical population genetics models to understand CUB.
Since the models developed in this work are based on mechanistic principles, observed patterns can be related directly to underlying biological mechanisms.
Thus, we have laid the groundwork upon which mechanistic models of various hypotheses can be simultaneously compared and evaluated.

\subsection{Consensus and disagreement}
While explanations for certain patterns of CUB are generally agreed upon, others are widely debated.
For instance, the work presented here as well as other previous studies suggests that CUB in genes with low expression is driven primarily by biased mutation rates \citep{ChamaryEtAl06,HershbergAndPetrov08,Subramanian08}.
This is due to the fact that in genes expressed at low levels, the efficacy of selection in driving CUB is weak.

However, in genes with high expression, patterns of CUB are thought be driven primarily by natural selection, although the nature of selection is debated.
For instance, selection for translation accuracy predicts that codons at sites that are evolutionarily conserved among proteins, will be better at minimizing missense errors than their coding synonyms \citep{Akashi94,AravaEtAl05,DrummondAndWilke08}.
This is because, evolutionarily conserved sites are thought to be functionally or structurally important and errors at these sites might render the protein nonfunctional.
Preference of codons with high tRNA abundances at these sites is thus thought to support this hypothesis.
However, as we show in Chapter \ref{ch:trna.codon}, the assumption that codons with high tRNA abundances lead to fewer errors is not always true and thus selection for translation accuracy is insufficient in explaining the presence of codons with high tRNA abundance at conserved sites \citep{ShahAndGilchrist10b}.

In addition, it has been observed that the codons at the start of a gene are either randomly distributed or have a higher proportion of suboptimal codons that the rest of the sequence.
The presence of slow or suboptimal codons at the beginning of a gene is thought to be adaptive for efficient ribosome queueing and prevention of collisions among ribosomes translating a given mRNA \citep{TullerEtAl10b}.
However, \citep{QinEtAl04,Gilchrist07,GilchristEtAl09} suggest that the presence of suboptimal codon at the beginning of a gene can also be explained by non-adaptive forces.
Selection against nonsense errors predicts that the degree of adaptation in coding sequences should increase along the length of a gene.
This is due to the fact that nonsense errors later in the sequence are more energetically expensive than earlier in the sequence as the cell has invested greater resources in making the polypeptide.
Since the cost of premature translation termination at the beginning of gene is relatively small, efficacy of selection in maintaining optimal codons may be weak.
In a study done with Drs. Michael Gilchrist and Russell Zaretzki, \citet{GilchristEtAl09} show that this is indeed the case and that the degree of adaptation in codon usage to minimize nonsense errors increases not only along the length of a gene but also with gene expression.

\section{Beyond translation}
Understanding the factors responsible for shaping patterns of codon usage provides important insights and estimates of processes affecting the fundamental process of protein translation.
However, insights gained from this understanding has far-reaching implications for a wide range of fields including that of epidemiology, systems biology and organismal and molecular evolution.

\subsection{Identifying genes under selection}
With the exponential growth in genomic data, it is now possible to identify the sets of genes that are under strong selection in various species.
Identifying these genes can allow us to make inferences about the organisms's environment as well as on its ecology.
For instance, the degree to which an aquatic organism expresses DNA UV repair pathways should reflect the amount of time it spends in the upper reaches of the water column \citep{BumaEtAl03}.

Traditionally the nature of selection acting on a gene - stabilizing or directional, is identified by comparing it with orthologues from its closely related species.
The ratio of non-synonymous to synonymous substitutions (dN/dS) in these sequences provides a measure of type of selection the gene is under \citep{NeiAndGojobori86,Yang98}.
If dN/dS$\ll1$, the sequence is thought be under stabilizing or purifying selection and if dN/dS$\gg1$, the sequence is thought be under positive or directional selection.
However, one of the fundamental assumptions made in this analysis is that synonymous substitutions are neutral.
As shown in this work, this is overly simplistic and could lead to various biases.
The work presented here allows us to quantify the strength of selection on synonymous codons of a sequence given its expression level and will help in defining better measures of selection.

An alternative to using dN/dS is using heuristic measures of codon usage bias (e.g. RSCU, CAI, $F_{op}$, E(g), $N_c$, CBI, CodonO, and RCB \citep{SharpAndLi87,Ikemura81,KarlinAndMrazek00,Wright90,BennetzenAndHall82,WanEtAl06}).
As mentioned earlier, a variety of heuristic measures have been developed to quantify the degree of bias in a statistical sense.
%Genes under selection (highly expressed genes) are generally thought to show stronger biases and sometimes in directions opposite to that observed in low expression genes.
In contrast to these heuristic measures, we have also developed an index of adaptation based on a specific biological process \citep{GilchristEtAl09}.
In any case, such measures allow us to identify genes that are under selection using the degree of bias observed in their codon patterns.

\subsection{Phylogenetic inference and codon bias}
One of the fundamental challenges in evolutionary biology is to understand the phylogenetic relationships among organisms.
In recent years, molecular data has replaced morphological traits in building phylogenetic trees \citep{JukesAndCantor69,Fink86,PosadaAndCrandall98}.
Models for building phylogenetic trees using gene sequences can be broadly classified into two categories - nucleotide based and codon based models \citep{GoldmanAndYang94}.
As the name suggests, nucleotide based models account for changes among sequences at the level of individual nucleotides by accounting for heterogeneity in mutations rates among various nucleotides.
Codon based models use codons as the fundamental unit of change when building trees rom multiple organisms.
In reality, codon based models are really amino acid based models as they account for changes in only those codons that lead to different amino acid.
This is generally done by penalizing codon substitutions based on the differences in properties of amino acids that are substituted.
As in the case of dN/dS, synonymous substitutions are generally thought to be neutral.
In contrast, along with Drs. Laura Kubatko (OSU) and Michael Gilchrist, I have worked on developing codon based models for phylogenetic inference that explicitly takes into account the effects of synonymous substitutions.
Such models would potentially provide greater resolution and lead to more accurate phylogenies.

\subsection{Codon usage and medicine}
A large number of sequenced organisms are pathogens.%\footnote{\url{http://www.ncbi.nlm.nih.gov/genomes/static/gpstat.html}}.
However, it is unlikely that our understanding of these organisms is ever going to rival that of model organisms.
For many of them, their sequence data might be the only source of information we may have for a while.
Thus by parsing genomic patterns such as those of codon usage in an evolutionary context can help us understand the biology of the organism.
For example, it has been shown that the patterns of codon usage in many viruses reflect an adaptation to the tRNA pools of their host \citep{ZhouEtAl99,PlotkinAndDushoff03,GroteEtAl05,ColemanEtAl08}.
Recently, \citep{ColemanEtAl08} showed that by changing only the codon usage of a virus genome but keeping the amino acid sequence same, one can dramatically reduce the infectivity of the virus.
Moreover, since the virus still produces the same proteins, albeit at a much lower rate, it elicits the same immune response and thus such modified viruses could be used for developing vaccines.



