\documentclass[11pt, letterpaper]{letter}

\signature{Cedric Landerer\\Ph.~D.~Candidate\\Department of Ecology \&\\Evolutionary Biology}
\address{University of Tennessee\\Department of Ecology \&\\Evolutionary Biology\\569 Dabney Hall\\Knoxville, TN 37996-1610}

\begin{document}
\begin{letter}{Editorial Staff\\Genome Biology and Evolution}



\opening{Dear Editor,}

On behalf of my co-authors, Dr.~ Brian C. O'Meara, Dr.~ Russel Zaretzki, Dr.~Michael Gilchrist, and myself, I would like to submit our article entitled ``Decomposing Mutation and Selection to Identify Mismatched Codon Usage'' for publication in \emph{Genome Biology and Evolution} as a \textbf{Research} article. 

Most studies on codon usage implicitly assume that the codon usage patterns of a genome is shaped by a single cellular environment. 
However, as species exchange genetic material, one would expect to see the influence of multiple cellular environments on a genomes codon usage pattern.
Given that transferred genes are likely to be less adapted than endogenous genes to their new cellular environment, we expect a greater genetic load of transferred genes if donor and recipient environment differ greatly in their selection bias, making such transfers less likely.
More practically, if differences in codon usage of transferred genes are unaccounted for, they may distort parameter estimates.
To illustrate these ideas, we analyze the CUB of the genome of \emph{Lachancea kluyveri} which has experienced a large introgression of exogenous genes.
The introgression replaced the left arm of the C chromosome and displays a $13 \%$ higher GC-content than the endogenous \emph{L. kluyveri} genome.
These characteristics make \emph{L. kluyveri} an ideal model to study mismatched codon usage.

We use ROC SEMPPR, a Bayesian population genetics model based on a mechanistic description of ribosome movement along an mRNA, to quantify the cellular environment in which genes have evolved by separately estimating the effects of mutation bias and selection bias on codon usage.
In contrast to often used heuristic approaches to study codon usage, ROC SEMPPR explicitly incorporates and distinguishes between mutation and selection effects on codon usage.
The separation of mutation and selection effects on codon usage not only allow us to describe two separate cellular environments shaping codon usage, it also allows us to identify the most likely source of the introgressed genes out of the 38 yeast lineages with sequenced genomes, estimate the age of the introgression, estimate the genetic load of these genes, both at the time of introgression and now, as well as make predictions about how the CUB of the introgressed genes will evolve in the future.
As a result, the work we present will be of great interest and use to the readers of GBE, including those interested the evolution of codon usage as well as researchers studying the transference of genomic material, and molecular adaptation.

Thank you for your time in this matter.
We look forward to hearing from you regarding this submission.

\closing{Sincerely,}

\end{letter}
\end{document}
