\documentclass[12pt,draft]{article}


\usepackage{lineno}
\usepackage{setspace}



\begin{document}
\doublespacing
\linenumbers


\section*{Introduction}

Organisms undergo changes in phenotype due to selection on fitness relevant traits in their current environment. 
The same principle applies to the genome adapting the intra-cellular environment.
If two 

\textit{Lachancea Kluyveri} is the earliest diverging lineage of the known \textit{Lachancea} clade. 
\textit{L. Kluyveri} experienced an introgression of 1Mb about XXX ago. 
The introgression shows a difference in average GC-content of \~ 13 \% compared to the rest if the \textit{L. Kluyveri}.

Codon usage is assumed to be associated with the available tRNA pool in a cell. 
Codons matching more abundant tRNA species are assumed to minimize ribosome pausing due to reduced waiting time for a correct tRNA. 
Differing codon usage bias in the introgressed region can either be caused by differences in mutation bias, relating to XXXX, or differences in selection for translation efficiency, indicate a shift in the available tRNA pool.
Differences in mutation bias should persist longer due to the absence of selection.   

We asked if the difference in GC-content is reflected in differences in codon usage bias.
We analyzed the two genomic regions in the \textit{L. Kluyveri} genome together and separately.
Our results indicate that the introgressed region shows a different signature of codon usage bias.
We employed a mechanistical model that allowed us to distinguish between effects of mutation bias, or bias in selection for translation efficiency.
Further inspection showed that mutation bias is dominating the observed difference, while differences in selection bias between codons is mostly in agreement between the two genomic regions. 

\begin{itemize}
	\item more general introduction, but into what? Introgression, Yeast, cub, adaptation, ...
	\item Introgression in Kluyveri, effects of introgressions, GC-content difference
	\item Codon Usage bias reflective of cellular environment, e.g. available tRNA pool
	\item Mechanistic model allows to distinguish between effects of selection and mutation on current observed state.
\end{itemize}
	
	
\section*{Results}

\begin{itemize}
	\item Question: GC-content variation causes difference in CUB, CUB difference driven by different selection or mutation environment of original host
	\item C-left has lower \% of high expression genes
	\item Observing variation in codon usage bias in kluyveri genome
	\item differences in mutation greater than in selection
	\item ignoring variation effects information extracted from the data
	\item no lachancia with similar cub known, most yeast have similar cub.
\end{itemize}

\section*{Discussion}

\begin{itemize}
	\item Faster decay of selection bias, versus mutation bias
	\item expected number of mutations, C-Left not at equilibrium
	\item low amount of high expression genes lead to longer persistence of GC-difference?
\end{itemize}


\end{document}