\documentclass[12pt]{article}

\usepackage{xspace}
\usepackage{lineno}
\usepackage{setspace}
\usepackage{graphicx}
\usepackage{subfigure}
\usepackage{float}
\usepackage{color}
\usepackage{caption}
\usepackage[margin=1in]{geometry}
\usepackage{epstopdf}
\usepackage{natbib}


\begin{document}
\doublespacing
\linenumbers



\noindent RH: LANDERER ET AL.--- Intragenomic variation in codon usage
% put in your own RH (running head)
% for POVs the RH is always POINT OF VIEW
\bigskip
\medskip
\begin{center}

% Insert your title:
\noindent{\Large \bf Predicting amino acid functionality from sequence data in a phylogenetic framework.}
\bigskip

% We don't use a special title page; the author information is entered
% like any other text.

% FOOTNOTES: We don't allow them in the manuscript, except in
% tables. Don't include any footnotes in the text.


\noindent{C\textsc{EDRIC} ~{L\textsc{ANDERER}}$^{1,2,*}$,
B\textsc{RIAN} C.~ {O\textsc{MEARA}}$^{1,2}$,
\textsc{AND}
M\textsc{ICHAEL} A.~{G\textsc{ILCHRIST}}$^{1,2}$}

\end{center}

\vfill

{\small
\noindent$^{1}$Department of Ecology \& Evolutionary Biology, University of Tennessee, Knoxville, TN 37996-1610\\
\noindent$^{2}$National Institute for Mathematical and Biological Synthesis, Knoxville, TN 37996-3410\\
\noindent$^{*}$Corresponding author. E-mail:~cedric.landerer@gmail.com
}

\vfill
\centerline{Version dated: \today}
\vfill
\newpage

\begin{abstract}
\end{abstract}	

\section*{Outline}
\subsection*{Introduction}
\begin{itemize}
	\item incorporating selection is important to for proper phylogenetic estimates, but where to get information about selection?	
	\item Default codon models do not include information on site specific amino acid preference, all amino acids are equally prefered.
	\item DMS or SelAC as options
	\item DMS experiments specific to stress factors, do not represent general evolution
	\begin{itemize}
		\item only applicable to genes which respond to environment that can be manipulated in the lab
		\item only applicable to organisms with fast gernarational turnover
		\item needs library of mutations, making mutations accesible that may not have a functional path (Maynard-Smith 60s)
	\end{itemize}
	\item SelAC estimates amino acid preferences from sequence data
	\begin{itemize}
		\item Depends on physico-chemical (PC) properties
		\item Rank of prefered amino acids is limited by PC
		\item What about observed frequencies?
	\end{itemize}
	\item Only comparing sequences (aa preferences), not models
\end{itemize}

\subsection*{Results}
\begin{itemize}
	\item Test various PC properties, can we even produce the observed rank?
	\item Compare DMS from Firnberg and Stiffler to SelAC and majority under SelAC and phydms
	\begin{itemize}
		\item Comparison of Frinberg under SelAC (three sequences: DMS, Majority, SelAC)
		\item Comparison of Frinberg under phydms (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under SelAC (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under phydms (three sequences: DMS, Majority, SelAC)
	\end{itemize}
	\item Compare AA preferences between the three sequences for both DMS sets
	\item Is there always a valid path from WT to prefered aa in DMS data?
\end{itemize}

\subsection*{Discussion}
\begin{itemize}
	\item What happends if PC can not produce observed rank? Still better than DMS?
\end{itemize}

\section*{Introduction}
\section*{Materials \& Methods}	
\section*{Results}
\section*{Discussion}

\section*{Supplemental Material}

\end{document}