\documentclass[12pt]{article}

\usepackage{xspace}
\usepackage{lineno}
\usepackage{setspace}
\usepackage{graphicx}
\usepackage{subfigure}
\usepackage{float}
\usepackage{color}
\usepackage{caption}
\usepackage[margin=1in]{geometry}
\usepackage{epstopdf}
\usepackage{natbib}


\begin{document}
\doublespacing
\linenumbers



\noindent RH: LANDERER ET AL.--- Estimating genetic load
% put in your own RH (running head)
% for POVs the RH is always POINT OF VIEW
\bigskip
\medskip
\begin{center}

% Insert your title:
\noindent{\Large \bf Estimating the genetic load of natural sequences in a phylogenetic framework.}
\bigskip

% We don't use a special title page; the author information is entered
% like any other text.

% FOOTNOTES: We don't allow them in the manuscript, except in
% tables. Don't include any footnotes in the text.
\begin{abstract}
\end{abstract}	



\noindent{C\textsc{EDRIC} ~{L\textsc{ANDERER}}$^{1,2,*}$,
B\textsc{RIAN} C.~ {O\textsc{MEARA}}$^{1,2}$,
\textsc{AND}
M\textsc{ICHAEL} A.~{G\textsc{ILCHRIST}}$^{1,2}$}

\end{center}

\vfill

{\small
\noindent$^{1}$Department of Ecology \& Evolutionary Biology, University of Tennessee, Knoxville, TN 37996-1610\\
\noindent$^{2}$National Institute for Mathematical and Biological Synthesis, Knoxville, TN 37996-3410\\
\noindent$^{*}$Corresponding author. E-mail:~cedric.landerer@gmail.com
}

\vfill
\centerline{Version dated: \today}
\vfill
\newpage
\section*{Introduction}
\begin{itemize}
	\item Genes evolve with natural selection favoring proteins that encode their function optimally
	\begin{itemize}
		\item To the extend at which the efficacy of selection becomes to weak and mutation and genetic drift pushes genes away from this optimum.
		\item Therefore, in the absence of any compromises between different selection pressures, mutation and genetic drift introduce a genetic load, reducing a proteins fitness. 
	\end{itemize}
	\item Genetic load is usually assessed relative to a predefined wilt-type.
	\begin{itemize}
		\item One could assess genetic load also relative to the genotype encoding a desired function most optimally.
		\item However, this requires to assess the fitness of each genotype. 
	\end{itemize}
	\item Previously deep mutation scanning (DMS) experiments have been utilized to assess site specific amino acid fitness for a variety of proteins.
	\begin{itemize}
		\item However, these experiments are limited to fast growing organisms that can be manipulated under laboratory conditions, and proteins where a specific selection pressure can be applied.
		\item Furthermore, DMS experiments utilize prepared libraries containing each genotype (ignoring epistatis), causing extremely low effective population sizes. 
		\item Thus, while mutation does not play a role, genetic drift reduces the efficacy of selection dramatically. 
	\end{itemize}
	\item We utilize SelAC, a phylogenetic framework, to assess the genetic load of naturally occurring sequence variation on the species level.
	\begin{itemize}
		\item SelAC is a mechanistic phylogenetic model rooted in population genetics, and estimates site specific selection from sequence data.
		\item SelAC does not assume a uniform stationary amino acid distribution across sites, thus allowing it to estimate the optimal amino acid for each position given the available sequence data.
		\item Furthermore, SelAC is not limited to  fast growing organisms that can be manipulated under laboratory conditions and thus applicable along the whole tree of life.
	\end{itemize}
	\item We predict the site specific optimal amino acid from sequence alignments of TEM, a $\beta$-lactamasein E. coli and cytochrome b (CytB), a mitochondrial transmembrane protein in whales.
	\item We then assess the genetic load of naturally occurring sequences TEM and CytB relative to the predicted functionally optimal amino acid sequence.
	\begin{itemize}
		\item We compare our genetic load estimates for TEM to empirical DMS estimates.
	\end{itemize}
	\item Furthermore, we will illustrate how the strength of selection varies along the analyzed proteins.
\end{itemize}

\section*{Results}
\begin{itemize}
	\item We predicted the functionally optimal amino acid at each site from the observed sequence variation using SelAC.
	\begin{itemize}
		\item TALK ABOUT OBSERVED SEQUENCE VARIATION
		\item We find that the predicted amino acid sequence has high agreement with the observed consensus sequence of the alignment (TEM: $99 \%$, CytB: $95 \%$).
		\item In contrast, the experimentally obtained sequence estimate only has an agreement of $49 \%$ with the observed TEM consensus sequence.
		\item Simulations based on the inferred optimal sequences showed that the we would not expect to the observed sequences to have evolved.
	\end{itemize}
	\item We assessed the genetic load the observed represent.
	\begin{itemize}
		\item The find that the genetic load of TEM differs greatly depending on the optimal amino acid sequence.
		\item 
	\end{itemize}

	\item Compare DMS from Firnberg and Stiffler to SelAC and majority under SelAC and phydms
	\begin{itemize}
		\item Comparison of Frinberg under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Frinberg under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
	\end{itemize}
\end{itemize}

\section*{Discussion}
\begin{itemize}
	\item SelAC sequence outperformes DMS experiments, reflecting evolution better than DMS sequences under artificial selection pressure.
	\item SelAC only uses prefered state as input, no information about 2nd or third prefered amino acid.
	\item The reduction of a DMS experiment to this state might be considered an unfair comparison, however, we tested the sequences under phydms (no reduction of information), with the same result.
	\item This also means that SelAC produces the same information a DMS experiment would, but for naturally evolving sequences and can be applied to any sequence.
	\item TEM/SHV have not evolved to combate specific human developed antibiotics, but as means of "warfare" between bacteria (need more reading here).
	\item This could be the cause for the great difference between DMS and observed sequences.
	\item SelAC, however can not provide any information about antibiotic resistency, making DMS very valuable, but not for phylogenetics. 
	\item but additional tip information could be combined with SelAC to get at this information (out of scope? future directions?).
\end{itemize}

\end{document}