\documentclass[12pt]{article}

\usepackage{xspace}
\usepackage{lineno}
\usepackage{setspace}
\usepackage{graphicx}
\usepackage{subfigure}
\usepackage{float}
\usepackage{color}
\usepackage{caption}
\usepackage[margin=1in]{geometry}
\usepackage{epstopdf}
\usepackage{natbib}


\begin{document}
\doublespacing
\linenumbers



\noindent RH: LANDERER ET AL.--- predicting amino acid functionality
% put in your own RH (running head)
% for POVs the RH is always POINT OF VIEW
\bigskip
\medskip
\begin{center}

% Insert your title:
\noindent{\Large \bf Predicting amino acid functionality from sequence data in a phylogenetic framework.}
\bigskip

% We don't use a special title page; the author information is entered
% like any other text.

% FOOTNOTES: We don't allow them in the manuscript, except in
% tables. Don't include any footnotes in the text.
\begin{abstract}
\end{abstract}	



\noindent{C\textsc{EDRIC} ~{L\textsc{ANDERER}}$^{1,2,*}$,
B\textsc{RIAN} C.~ {O\textsc{MEARA}}$^{1,2}$,
\textsc{AND}
M\textsc{ICHAEL} A.~{G\textsc{ILCHRIST}}$^{1,2}$}

\end{center}

\vfill

{\small
\noindent$^{1}$Department of Ecology \& Evolutionary Biology, University of Tennessee, Knoxville, TN 37996-1610\\
\noindent$^{2}$National Institute for Mathematical and Biological Synthesis, Knoxville, TN 37996-3410\\
\noindent$^{*}$Corresponding author. E-mail:~cedric.landerer@gmail.com
}

\vfill
\centerline{Version dated: \today}
\vfill
\newpage
\section*{Introduction}
\begin{itemize}
	\item Understanding selection pressure is one of the fundamental goals of evolution
	\begin{itemize}
		\item To that extend, a lot of theoretical work and experimental procedures have been developed.
	\end{itgemize}


	\item The introduction of selection into phylogenetic frameworks has been a long going effort.
	\begin{itemize}
		\item Many models have been developed (Yang and Nielsen, Halpern and Bruno, ...)
		\item Insert brief review of methods.
		\begin{itemize}
			\item Models provided great theoretical inside.
			\item Still often assume uniform stationary distribution of amino acids across sites.
		\end{itemize}
	\end{itemize}
	\item So far these models find limited application as these frameworks are very parameter rich.
	\begin{itemize}
		\item The most popular models/tools, however, are still based purely on the mutation process (RaxML, RevBayes). 
	\end{itemize}
	\item A recent take on the incorporation of selection on a protein is the independent estimation of fitness effects.
	\begin{itemize}
		\item Deep Mutation Scanning (DMS) experiments provide site specific fitness values on synonymous and non-synonymous mutations (focus on non-synonymous).
		\begin{itemize}
			\item This limits the number of estimated parameters greatly and allows for computationally feasible models.
		\end{itemize}
		\item However, the information on selection gained by DMS experiments is limited to single proteins of organisms that can be manipulated in the laboratory with short generation times.
		\begin{itemize}
			\item This greatly limits its application in phylogenetics
		\end{itemize}
	\end{itemize}
	\item SelAC is a mechanistic model that utilizes the idea of site specific selection, and estimates it from sequence data.
	\begin{itemize}
		\item SelAC has multiple advantages over other methods incorporating selection.
		\begin{itemize}
			\item It does not assume a uniform stationary amino acid distribution across sites.
			\item Does not depend on experimental data, and can therefore be applied to all codon sequences.
			\item Clearly states model assumption and provides interpretable parameter estimates beyond branch length and nucleotide transition rates.
		\end{itemize}
		\item Due to SelACs hierarchical model structure it can also be parameterized with relatively few parameters.
	\end{itemize}
	\item In this study, we compare the quality of phylogenetic estimates obtained utilizing DMS experiments to estimates from SelAC.
	\begin{itemize}
		\item We utilize DMS experiments from Firnberg (2014) and Stifler (2016) for the TEM $\beta$-lactamase of \textit{E. coli}.
		\item We use phydms, a tool explicitly designed to utilize selection information from DMS experiments for an independent assessment of the SelAC estimated stationary amino acid distribution. 
		\item We compare model fit and adequacy of the DMS and SelAC amino acid preferences using SelAC and phydms.
		\begin{itemize}
			\item We show that DMS experiments can have trouble accurately reflecting natural evolution of protein sequences.	
			\item We find that amino acid preferences estimated with SelAC provides better model adequacy than DMS experiments.
			\item We show that information about amino acid preference can be extracted from sequence data using SelAC.
		\end{itemize}	
	\end{itemize}
\end{itemize}

\section*{Results}
\begin{itemize}
	\item Compare DMS from Firnberg and Stiffler to SelAC and majority under SelAC and phydms
	\begin{itemize}
		\item Comparison of Frinberg under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Frinberg under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
	\end{itemize}
	\item Comparison of perfered sequence
	\begin{itemize}
		\item Simulations of sequences under each prefered sequence.
		\item Only majority rule (duh) and SelAC agree with observed sequences. 
	\end{itemize} 
	\item SelAC is dependent on choice of PC propertie to produce amino acid rankorder and assumes stabilizing selection. 
	\begin{itemize}
		\item Rankorder of certain sites can not be produced by any of the PC checked (no combination checked)
	\end{itemize}
\end{itemize}

\section*{Discussion}
\begin{itemize}
	\item SelAC sequence outperformes DMS experiments, reflecting evolution better than DMS sequences under artificial selection pressure.
	\item SelAC only uses prefered state as input, no information about 2nd or third prefered amino acid.
	\item The reduction of a DMS experiment to this state might be considered an unfair comparison, however, we tested the sequences under phydms (no reduction of information), with the same result.
	\item This also means that SelAC produces the same information a DMS experiment would, but for naturally evolving sequences and can be applied to any sequence.
	\item TEM/SHV have not evolved to combate specific human developed antibiotics, but as means of "warfare" between bacteria (need more reading here).
	\item This could be the cause for the great difference between DMS and observed sequences.
	\item SelAC, however can not provide any information about antibiotic resistency, making DMS very valuable, but not for phylogenetics. 
	\item but additional tip information could be combined with SelAC to get at this information (out of scope? future directions?).
\end{itemize}

\end{document}