\documentclass[12pt]{article}

\usepackage{xspace}
\usepackage{lineno}
\usepackage{setspace}
\usepackage{graphicx}
\usepackage{subfigure}
\usepackage{float}
\usepackage{color}
\usepackage{caption}
\usepackage[margin=1in]{geometry}
\usepackage{epstopdf}
\usepackage{natbib}


\begin{document}
\doublespacing
\linenumbers



\noindent RH: LANDERER ET AL.--- predicting amino acid functionality
% put in your own RH (running head)
% for POVs the RH is always POINT OF VIEW
\bigskip
\medskip
\begin{center}

% Insert your title:
\noindent{\Large \bf Predicting amino acid functionality from sequence data in a phylogenetic framework.}
\bigskip

% We don't use a special title page; the author information is entered
% like any other text.

% FOOTNOTES: We don't allow them in the manuscript, except in
% tables. Don't include any footnotes in the text.
\begin{abstract}
\end{abstract}	



\noindent{C\textsc{EDRIC} ~{L\textsc{ANDERER}}$^{1,2,*}$,
B\textsc{RIAN} C.~ {O\textsc{MEARA}}$^{1,2}$,
\textsc{AND}
M\textsc{ICHAEL} A.~{G\textsc{ILCHRIST}}$^{1,2}$}

\end{center}

\vfill

{\small
\noindent$^{1}$Department of Ecology \& Evolutionary Biology, University of Tennessee, Knoxville, TN 37996-1610\\
\noindent$^{2}$National Institute for Mathematical and Biological Synthesis, Knoxville, TN 37996-3410\\
\noindent$^{*}$Corresponding author. E-mail:~cedric.landerer@gmail.com
}

\vfill
\centerline{Version dated: \today}
\vfill
\newpage
\section*{Outline}
\subsection*{Introduction}
\begin{itemize}
	\item Classic phylogenetic approaches are designed to emulate the mutation of sequences and infere the time it takes to mutate from one to another as well as the relationship of sequences to oneanother.
	\item However, these models still often ignore the effects mutations have on a protein sequence and how this effect in turn affects the fixation probability due to differences in fitness.
	\item Incorporating selection into phylogenetic analysis is therefore important and many attempts have been made to do so (Yang \& Nilesen, Halpern \& Bruno, ...) 
	\item These codon models do not include information on site specific amino acid preference, all amino acids are equally prefered at each site (staionary distribution is uniform).
	\item ????How to go from here without copying SelAC introduction????
	\item Novel approaches attempt to incorporate information about site specific amino acid fitness obtained from deep mutation scanning (DMS) experiments.
	\item These models allow the incorporation of amino acid preference and estimate how preferences in natural sequences deviate from experimentally obtained preferences. 
	\item While there is great value in DMS experiments and it has been shown that DMS informed models are superior to classical codon models (Bloom) these experiments are limited to singele proteins that can be put under artificial selection in the lab.
	\item SelAC in turn estiamtes the functionality of each amino acid at each site based on a mechanistical model of protein selection.
	\item SelAC has the advaantage that:
	\begin{itemize}
		\item in contrast to other codon models does not assume a uniform amino acid preference at each site.
		\item it  estimates the amino acid preference at each site from the sequence data and therefore is not limited to single proteins, or proteins that can be experimentally modified.
		\item is not limited to fast growing lab cultivated organisms.
		\item does not requiere experimental overhead (mutation libray, artificial selection, ...)
		\item no artificial stress factors (selection pressure), but sequences that have evolved naturally.
	\end{itemize}
	\item In this work, we check compare the SelAC estimated amino acid preference with amino acid prefernces obtained by DMS approaches and a simple majority estimate under two models, SelAC itself, and phydms (Hilton 2017)
	\begin{itemize}
		\item phydms was specifically developed to work with data from DMS experiments, using experimental amino acid preferences.
		\item SelAC estimates amino acid preferences based on physico-chemical (PC) properties and distances between amino acids and an infered prefered amino acid
		\item We converted SelAC and majority rule amino acid preferences into a format that works with phydms and checked which sequence performes best under each model.
	\end{itemize}
\end{itemize}

\subsection*{Results}
\begin{itemize}
	\item Compare DMS from Firnberg and Stiffler to SelAC and majority under SelAC and phydms
	\begin{itemize}
		\item Comparison of Frinberg under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Frinberg under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under SelAC for TEM and SHV (three sequences: DMS, Majority, SelAC)
		\item Comparison of Stiffler under phydms for TEM and SHV (three sequences: DMS, Majority, SelAC)
	\end{itemize}
	\item Comparison of perfered sequence
	\begin{itemize}
		\item Simulations of sequences under each prefered sequence.
		\item Only majority rule (duh) and SelAC agree with observed sequences. 
	\end{itemize} 
	\item SelAC is dependent on choice of PC propertie to produce amino acid rankorder and assumes stabilizing selection. 
	\begin{itemize}
		\item Rankorder of certain sites can not be produced by any of the PC checked (no combination checked)
	\end{itemize}
\end{itemize}

\subsection*{Discussion}
\begin{itemize}
	\item SelAC sequence outperformes DMS experiments, reflecting evolution better than DMS sequences under artificial selection pressure.
	\item SelAC only uses prefered state as input, no information about 2nd or third prefered amino acid.
	\item The reduction of a DMS experiment to this state might be considered an unfair comparison, however, we tested the sequences under phydms (no reduction of information), with the same result.
	\item This also means that SelAC produces the same information a DMS experiment would, but for naturally evolving sequences and can be applied to any sequence.
	\item TEM/SHV have not evolved to combate specific human developed antibiotics, but as means of "warfare" between bacteria (need more reading here).
	\item This could be the cause for the great difference between DMS and observed sequences.
	\item SelAC, however can not provide any information about antibiotic resistency, making DMS very valuable, but not for phylogenetics. 
	\item but additional tip information could be combined with SelAC to get at this information (out of scope? future directions?).
\end{itemize}

\section*{Introduction}
\section*{Materials \& Methods}	
\section*{Results}
\section*{Discussion}

\section*{Supplemental Material}

\end{document}